examples/README.md

```diff
@@ -193,64 +193,57 @@ For this example the following is needed in the selected region:
 ### [image-builder.yaml] ![core-badge]
 
 This Blueprint uses the [Packer template module][pkr] to create custom VM images
-by applying software and configurations to existing images.
+by applying software and configurations to existing images. This example takes
+the following steps:
 
-This example performs the following:
+1. Creates a network with outbound internet access in which to build the image (see
+[Custom Network](#custom-network-deployment-group-1)).
+2. Creates a script that will be used to customize the image (see
+[Toolkit Runners](#toolkit-runners-deployment-group-1)).
+3. Builds a custom Slurm image by executing the script on a standard Slurm image
+(see [Packer Template](#packer-template-deployment-group-2)).
+4. Deploys a Slurm cluster using the custom image (see
+[Slurm Cluster Based on Custom Image](#slurm-cluster-based-on-custom-image-deployment-group-3)).
 
-1. Creates a network needed to build the image (see
-   [Custom Network](#custom-network-deployment-group-1)).
-2. Sets up a script that will be used to configure the image (see
-   [Toolkit Runners](#toolkit-runners-deployment-group-1)).
-3. Builds a new image by modifying the Slurm image (see
-   [Packer Template](#packer-template-deployment-group-2)).
-4. Deploys a Slurm cluster using the newly built image (see
-   [Slurm Cluster Based on Custom Image](#slurm-cluster-based-on-custom-image-deployment-group-3)).
+Create the deployment folder from the blueprint:
```

Comment:
nit: In the updated version it is less obvious where instructions begin/end vs other material. Consider adding a minor header with something like "deployment instructions".

Reply:
Done with some improved text.